subgraphs/rocket-pool/package.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
can you make this file look similar to this one: https://github.com/messari/subgraphs/blob/master/subgraphs/_reference_/package.json?

And then run `npm run format`?